### PR TITLE
Diverse Skyrim - Battle of Whiterun Fix

### PR DESCRIPTION
**Pull request by TateTaylorOH**
Adds a suggestion for the User to download [Diverse Skyrim - Battle of Whiterun Fix](https://www.nexusmods.com/skyrimspecialedition/mods/47939) if they are using Diverse Skyrim.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -6384,6 +6384,11 @@ plugins:
     tag:
       - Delev
       - Relev
+    msg:
+      - type: say
+        content: 'It is recommended that you use [Diverse Skyrim - Battle of Whiterun Fix](https://www.nexusmods.com/skyrimspecialedition/mods/47939) with this mod.'
+        lang: eng
```
Comment (sibir-ine) [inline masterlist.yaml]: @PierreDespereaux The message can be modified to fit a boilerplate message that already exists, so the end result will be: "It's highly recommended that you also use [Diverse Skyrim - Battle of Whiterun Fix](https://www.nexusmods.com/skyrimspecialedition/mods/47939)."

Also, that language key shouldn't be there.

```suggestion
      - <<: *alsoUseX
        subs: [ '[Diverse Skyrim - Battle of Whiterun Fix](https://www.nexusmods.com/skyrimspecialedition/mods/47939)' ]
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]

**Review by sibir-ine — APPROVED** [review]

**Comment by sibir-ine** [comment]
@PierreDespereaux Thanks!

**Comment by TateTaylorOH** [comment]
No, thank you :)

**Comment by MacSplody** [comment]
A similar fix may be included in Immersive Citizens.
image

**Comment by sibir-ine** [comment]
> A similar fix may be included in Immersive Citizens.

It indeed is. I'll add a message for the fix & update this message appropriately.

**Pull request merged**